addon.json

```diff
@@ -0,0 +1,66 @@
+{
+    "identifier": "com.seeq.addon.correlation",
+    "name": "Correlation Analysis",
+    "description": "Determine cross correlations and time shifts to maximize correlations among signals",
+    "version": "Input version here",
+    "license": "Apache-2.0 license",
+    "icon": "fa fa-th",
+    "tags": {"maintainer":"Seeq"},
```

Comment:
This should be a top level property in the file: `"maintainer": "Seeq",`. I noticed it was specified incorrectly in our docs. I've updated it: https://seeq.atlassian.net/wiki/spaces/SD/pages/2835480688/Elements.